electron-app/src/components/inputs/AssetSelect.vue

```diff
@@ -48,6 +49,15 @@ const { mapState } = createNamespacedHelpers('balances');
 export default class AssetSelect extends Vue {
   supportedAssets!: SupportedAsset[];
 
+  @Prop({ required: false, default: null })
+  items?: string[];
```

Comment:
```suggestion
  items!: string[] | null;
```
There is no need to make it optional since it would never be `undefined`. It will always be either a `string[]` or `null`

Reply:
Clear.